internal/mode/static/nginx/config/tracing_template.go

```diff
@@ -0,0 +1,12 @@
+package config
+
+var otelTemplateText = `
+otel_exporter {
+    endpoint {{.ExporterEndpoint}};
+    interval {{.Interval}};
+    batch_size  {{.BatchSize}};
+    batch_count {{.BatchCount}};
+}
+otel_trace {{if .Enabled}}on{{else}}off{{end}};
```

Comment:
Fair point, but I'm not sure how this will work without enabling it in the internal locations. Right now, this is what tracing looks like following the advanced routes example: 
<img width="1263" alt="Screenshot 2023-11-15 at 16 15 53" src="https://github.com/nginxinc/nginx-gateway-fabric/assets/18287516/05790401-0230-4e79-80e2-62985647c155">

I'll do some more testing.

Reply:
Hmm, yeah that may be a bit confusing if users see these internal paths but not the top level ones.

Reply:
what if enable only in internal locations, but customize the name to be the name of the parent location using https://nginx.org/en/docs/ngx_otel_module.html#otel_span_name ?